src/repositories/logRepository.test.ts

```diff
@@ -0,0 +1,175 @@
+import { findOrganizationById } from './organizationRepository';
+import {
+  createAccessTokenForTest,
+  SEED_DATA,
+  serverUrl,
+} from '../../test/testUtils';
+import { assert } from 'chai';
+import { Log, LogStatus } from '../entities/log';
+import { generateRandomString } from '../utils/utils';
+import {
+  createNewLog,
+  updateAccessTokenAndApplication,
+  updateFailedLog,
+  updateScopeLog,
+  updateSuccessLog,
+} from './logRepository';
+import { scopeLabels } from '../services/scopeService';
+import axios from 'axios';
+import { createBasicAuthentication } from '../utils/authorizationUtils';
+import { findApplicationById } from './applicationRepository';
+import { Application } from '../entities/application';
+
+describe('createNewLog() test cases', () => {
+  it('should createNewLog', async () => {
+    const trackId = `${new Date().getTime()}_${generateRandomString(6)}`;
+    const logData = {
+      url: 'http://google.come',
+      status: LogStatus.PENDING,
+      trackId,
+      method: 'GET',
+    };
+    const log = await createNewLog(logData);
+    assert.isOk(log);
+    assert.equal(log.trackId, trackId);
+  });
+});
+
+describe('updateScopeLog() test cases', () => {
+  it('should updateScopeLog', async () => {
+    const trackId = `${new Date().getTime()}_${generateRandomString(6)}`;
+    const logData = {
+      url: 'http://google.come',
+      status: LogStatus.PENDING,
+      trackId,
+      method: 'GET',
+    };
+    const log = await createNewLog(logData);
+    await updateScopeLog({
+      trackId,
+      scope: scopeLabels.CREATE_DONATION,
+    });
+    const newLog = await Log.findOne({ where: { id: log.id } });
+    assert.isOk(newLog);
+    assert.equal(newLog?.scope, scopeLabels.CREATE_DONATION);
+  });
+  it('should not updateScopeLog because log not found', async () => {
+    const trackId = `${new Date().getTime()}_${generateRandomString(6)}`;
+    await updateScopeLog({
```

Comment:
Same for the updatedAt Column lets save it in it's own column too. Since we are updating logs.